vulkaninfo/vulkaninfo.c

```diff
@@ -5500,6 +5500,10 @@ int main(int argc, char **argv) {
 
     if (html_output) {
         out = fopen("vulkaninfo.html", "w");
+        if (!out) {
+            printf("Unable to open vulkaninfo.html for writing\n");
+            return 1;
```

Comment:
Not sure if this is conventional in this file.
Typically, if error happens `ERR_EXIT` is called with some fake `VkResult` code (e.g. if `malloc` fails `ERR_EXIT(VK_ERROR_OUT_OF_HOST_MEMORY)` is called).

Reply:
The previous usage failures, do a return 1; also not sure we have a return code for can't open file.

Reply:
It does not matter. `VK_ERROR_INITIALIZATION_FAILED` or `VK_ERROR_MEMORY_MAP_FAILED` feels vaguely appropriate.
`return` too is ok I guess. Just have to be kept in mind were someone to refactor the error handling to support non-Vulkan errors....